phpBB/phpbb/db/migration/tool/module.php

```diff
@@ -262,7 +262,7 @@ public function add($class, $parent = 0, $data = array())
 
 			// Success
 			$module_log_name = ((isset($this->user->lang[$data['module_langname']])) ? $this->user->lang[$data['module_langname']] : $data['module_langname']);
-			$phpbb_log->add('admin', $user->data['user_id'], $user->ip, 'LOG_MODULE_ADD', false, array($module_log_name));
+			$phpbb_log->add('admin', ANONYMOUS, $user->ip, 'LOG_MODULE_ADD', false, array($module_log_name));
```

Comment:
I'd prefer checking if user id exists and use it if it is. Or will this always be undefined?